### ISSUE TITLE
not accessible with screen readers

### ISSUE THREAD

**Issue by MesterPerfect** [iXab3r/MicSwitch#86]
hi
I'm a blind user
I can't use MicSwitch with screen readers
Because it does not support accessibility
Can you fix that?

**Comment by iXab3r** [comment]
hi, will definitely look into it tomorrow

**Comment by MesterPerfect** [comment]
> hi, will definitely look into it tomorrow

ok

**Comment by iXab3r** [comment]
Added support for NVDA and Screen Narrator in version 228
There is a known issue with Screen Narrator and .NET 6 - textboxes report "Nothing in view" instead of actual text. Will look for workaround, but not sure that it's fixable on application side

Please report if anything works not as you expect or is no convenient

**Comment by MesterPerfect** [comment]
> Added support for NVDA and Screen Narrator in version 228 There is a known issue with Screen Narrator and .NET 6 - textboxes report "Nothing in view" instead of actual text. Will look for workaround, but not sure that it's fixable on application side
> 
> Please report if anything works not as you expect or is no convenient

hi 
Thank you for the quick response in support of accessibility with screen readers
The program is much better

There are two issues

Regarding the program icon in the system tray
I can't activate its options using applications key on keyboard

second issues
The screen reader cannot read names of items in Context menu

![1](https://user-images.githubusercontent.com/44323153/168451869-2307a452-c7e3-4098-aab8-d750713ac624.jpg)
![2](https://user-images.githubusercontent.com/44323153/168451871-fd57089f-82ea-4699-8b73-e9efc8f6944a.jpg)


Whenever a menu is activated, All items are pronounced like this 

PoeShared.Audio.ViewModels.NotificationTypeWrapperViewModel
PoeShared.Audio.ViewModels.NotificationTypeWrapperViewModel
PoeShared.Audio.ViewModels.NotificationTypeWrapperViewModel

**Comment by iXab3r** [comment]
Apologies for not answering for so long - I've been in process of relocation from one country to another. Will look into it asap

**Comment by MesterPerfect** [comment]
> Apologies for not answering for so long - I've been in process of relocation from one country to another. Will look into it asap

No problem, good luck in your new house
